internal/bzlmod/go_deps.bzl

```diff
@@ -144,7 +144,7 @@ def _get_override_or_default(specific_overrides, gazelle_default_attributes, def
     return default_value
 
 def _get_directives(path, gazelle_overrides, gazelle_default_attributes):
-    return _get_override_or_default(gazelle_overrides, gazelle_default_attributes, DEFAULT_DIRECTIVES_BY_PATH, path, [], "directives")
+    return _get_override_or_default(gazelle_overrides, gazelle_default_attributes, DEFAULT_DIRECTIVES_BY_PATH, path, ["gazelle:proto disable_global"], "directives")
```

Comment:
Do we want this to be `disable` or `disable_global` ? (I'll poke around to understand the difference better)

Reply:
For a Go module without any proto file, `disable` and `disable_global` no longer have a difference. I slightly prefer "disable" just because it's shorter and doesn't let people wonder what "global" means